cmd/snap/cmd_info.go

```diff
@@ -119,52 +121,22 @@ func maybePrintBase(w io.Writer, base string, verbose bool) {
 	}
 }
 
-func tryDirect(w io.Writer, path string, verbose bool, termWidth int) bool {
-	path = norm(path)
-
+func loadDirect(path string) (*client.Snap, error) {
```

Comment:
load seems like the wrong verb

Reply:
examine?

Reply:
I'm not sure what's wrong with `load`, but `examine` is probably wrong

Reply:
as we discussed  "load" sounds like ti will load the entire snap in memory, and we tend not to use it.

clientSnapFromSnapFile is probably the most verbose name I can think for this?

Reply:
going with `clientSnapFromPath` (because `SnapFile` is arguably what `snap.Open` gives you).